src/main/java/microsoft/exchange/webservices/data/AttachmentCollection.java

```diff
@@ -372,15 +372,17 @@ public void validate() throws Exception {
             && this.owner.getService().getRequestedServerVersion()
             .ordinal() >= ExchangeVersion.Exchange2010_SP2
             .ordinal()) {
-          FileAttachment fileAttachment = (FileAttachment) attachment;
+          if (attachment instanceof FileAttachment) {
+            FileAttachment fileAttachment = (FileAttachment) attachment;
 
-          if (fileAttachment.isContactPhoto()) {
-            if (contactPhotoFound) {
-              throw new ServiceValidationException(
-                  Strings.MultipleContactPhotosInAttachment);
-            }
+            if (fileAttachment.isContactPhoto()) {
+              if (contactPhotoFound) {
```

Comment:
looking at the code I was wondering if this is a contradiction because `contactPhotoFound` is set to false and only changed to true if it is already true.